GoMeet/src/main/webapp/js/fetch-wrapper.js

```diff
@@ -1,4 +1,4 @@
-/** Wraps the fetch function. */
+/** Data accesss object for meeting locations */
 class FetchWrapper {
```

Comment:
nit: Maybe rename the class and file to MeetingLocationDAO and meeting-location-dao.js so that it's consistent with Anna and I's DAOs

Reply:
Sorry my bad - you've already done this in another commit!